cluster/gce/gci/mounter/mounter.go

```diff
@@ -71,14 +72,17 @@ func mountInChroot(rootfsPath string, args []string) error {
 		return nil
 	}
 
-	if !strings.EqualFold(string(output), nfsRPCBindErrMsg) {
+	if !strings.EqualFold(string(output), nfsRPCBindNotRunningErrMsg) {
 		// Mount failed but not because of RPC bind error
 		return fmt.Errorf("mount failed: %v\nMounting command: %s\nMounting arguments: %v\nOutput: %s", err, chrootCmd, args, string(output))
 	}
 
 	// Mount failed because it is NFS V3 and we need to run rpcBind
 	output, err = exec.Command(chrootCmd, rootfsPath, rpcBindCmd, "-w").CombinedOutput()
-	if err != nil {
+	// There is a race condition where multiple pods are scheduled on the same node and the rpcbind command is executed
+	// at the same time, which causes the unexpected setup failure for the second and subsequent pods.
+	// Let's ignore this error and try to mount again.
+	if err != nil && !strings.EqualFold(string(output), nfsRPCBindAlreadyRunningErrMsg) {
```

Comment:
As a quick fix this seems to address the problem. I'm just wondering about the retry logic: it retries once. Wouldn't it be better to retry in a loop?

Reply:
kubelet will retry the mount operation if the operation is failed. However, the rpcbind might not be started when the mount operation is triggered, so the custom mounter start the rpcbind and then redo mount operation. The second mount operation should be succeed if no unexpected condition happens.